api/v1/handlers.go

```diff
@@ -5098,52 +5031,17 @@ func sanitizePath(p string) (string, error) {
 func (s *apiV1) handleColfsAdd(c echo.Context, u *util.User) error {
 	coluuid := c.QueryParam("coluuid")
 	contid := c.QueryParam("content")
-	npath := c.QueryParam("path")
```

Comment:
honestly, we should be using `path`. It is the right word for what is being done. Collection API v2 should change `dir` to `path`

Reply:
These changes are on the v1 file, not v2